### PR TITLE
Integrate C1 (customer support) script

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Integrate customer first (C1) script globally on the site.

Only show widget on-demand.

Remove intercom stuff.

## Justify why they are needed

We wanna test this out in staging to begin with.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Review by alebedev — APPROVED** [review]
Looks good.  I don't think we need separate feature flag when we can just set script URL to empty

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/services/CustomerFirst.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  )
+}
+
+export const useCustomerFirst = () => {
+  const setOpen = useSetAtom(OPEN_ATOM)
+
+  useEffect(() => {
+    const handleMessage = ({ data }: MessageEvent<MessageData>) => {
+      if (data === 'showWidget') setOpen(true)
+      else if (data === 'hideWidget') setOpen(false)
```
Comment (alebedev) [inline apps/store/src/services/CustomerFirst.tsx:29]: Do we get `hideWidget` from C1?

**Review by gustaveen — APPROVED** [review]
Nice! 🙌 
Should we try to build our own floating button instead to make it easier to control? :) 
Saw that clicking the "Telefonnummer" button in the widget goes to this [page](https://new.hedvig.com/se/support/inte-medlem), is that intentional?

**Comment by robinandeer** [comment]
> Nice! 🙌 
> Should we try to build our own floating button instead to make it easier to control? :) 
> Saw that clicking the "Telefonnummer" button in the widget goes to this [page](https://new.hedvig.com/se/support/inte-medlem), is that intentional? 

Hmm that's strange --- not sure, but hopefully Malin is on top of that

We could maybe build our own button but I think we can influence how it looks but discussing with them to begin with.

**Pull request merged**